### PR TITLE
Update JSDocs, linter and dependencies

### LINKED ISSUE #33: Switch to eslint
For [gts](https://github.com/google/gts) compatibility, switch back to `eslint`

### LINKED ISSUE #40: Update to Local Home SDK 1.1.0
Should be a small change but should update LHSDK dependency to reflect the latest release

### PR DESCRIPTION
**Pull request by cdaly333**
This PR is mostly stylistic changes.  Formatted a lot of comments that were excessively long and switched back to `eslint` in compliance with `gts`.  I also updated UDP response spec/functionality to comply with LHSDK 1.1.0. 

**Changes:**
- Update to `eslint` from `tslint`
- Fix `eslint` rule violations
- Minor tweaks to `package.json`
- Fix some comment punctuation and phrasing (add periods to all comments)
- Shorten comments to more closely adhere to 80 character line limit
- Update for compatibility with LHSDK 1.1.0

**Issues:**
- resolves #33 
- resolves #40

### CODE REVIEW

File: src/platform/mock-local-home-platform.ts [hunk 1]
```diff
@@ -100,10 +102,11 @@ export class MockLocalHomePlatform {
   }
 
   /**
-   * Forms an `ExecuteRequest` with the given commands and passes it to the fulfillment app.
+   * Forms an `ExecuteRequest` with the given commands.
+   * Passes it to the fulfillment app.
    * @param requestId  The request id to set in the `ExecuteRequest`
```
Comment (Fleker) [inline src/platform/mock-local-home-platform.ts:107]: Is the plan to put periods at the end of each line? This is one line without a period.

File: test/platform/test-device-manager.ts [hunk 2]
```diff
@@ -20,7 +20,10 @@ const COMMAND_REQUEST = createUdpDeviceCommand(
  * Returns a simple execute request used for testing
  * @param command A command to set in the sample request
```
Comment (Fleker) [inline test/platform/test-device-manager.ts:22]: Should add a `@param params` here and a `@returns`.

File: src/platform/mock-local-home-platform.ts (hunk 1 above)
Comment (cdaly333) [reply src/platform/mock-local-home-platform.ts:113]: Yes.  Done. I noticed the Local Home SDK has periods at the end of all JSDocs so am going to generally follow that rule.  Went through and added periods to all comments - should be fairly consistent now.

File: test/platform/test-device-manager.ts (hunk 2 above)
Comment (cdaly333) [reply test/platform/test-device-manager.ts]: Done.

File: test/platform/test-platform-execute.ts [hunk 3]
```diff
@@ -70,18 +71,20 @@ test('execute-handler-command-success', async t => {
...
   stubs.deviceManagerStub.addExpectedCommand(
     validCommand,
     new UdpResponseData(EXECUTE_REQUEST_ID, DEVICE_ID, new UdpResponse())
   );
 
   const executeCommands = createSimpleExecuteCommands(
     DEVICE_ID,
-    'action.devices.commands.OnOff'
+    'actions.devices.commands.OnOff',
+    {on: true},
```
Comment (Fleker) [inline test/platform/test-platform-execute.ts:83]: OnOff would only have a parameter of `on` and not a color.

File: test/platform/test-platform-execute.ts [hunk 4]
```diff
@@ -70,18 +71,20 @@ test('execute-handler-command-success', async t => {
...
+    {color: 'red'}
```
Comment (Fleker) [inline test/platform/test-platform-execute.ts:84]: This seems like a command param and not custom data. Can you adopt something closer to a `foo: 'bar'` like in the documentation examples?

File: test/platform/test-platform-execute.ts (hunk 3 above)
Comment (cdaly333) [reply test/platform/test-platform-execute.ts:83]: Resolved.

File: test/platform/test-platform-execute.ts (hunk 4 above)
Comment (cdaly333) [reply test/platform/test-platform-execute.ts]: Replaced with [documentation example](https://developers.google.com/assistant/smarthome/reference/intent/execute).  Also updated example tests.

**Comment by Fleker** [comment]
LGTM

**Pull request merged**